app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -49,7 +49,7 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => {
         }
       }}
     >
-      {({ experiment, analysis }) => {
+      {({ experiment, refetch, analysis }) => {
```

Comment:
Is this still needed?

Reply:
Ah, yeah, no I think that's a leftover from moving the component. Will delete